packages/angular_bloc/example/example.dart

```diff
@@ -1,69 +0,0 @@
-import 'dart:async';
```

Comment:
I don't think we should delete this file because it negatively impacts the pub score and is [displayed directly on pub.dev](https://pub.dev/packages/angular_bloc#-example-tab-)

Reply:
Oh good to know, will revert it and leave a comment then. Thanks Felix!